modules/calypso-redirects.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+if ( ! class_exists( 'Calypso_Redirects' ) ) :
+
+class Calypso_Redirects {
+
+	public function __construct() {
+		if ( ! get_option( 'jetpack_calypso_redirects_active' ) || ! is_admin() ) {
+			return;
+		}
+
+		add_action( 'admin_menu', array( $this, 'menu_redirects' ), 30 );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+
+		add_filter( 'allowed_redirect_hosts', array( $this, 'allow_calypso_domain' ), 10, 1 );
+
+		add_action( 'load-toplevel_page_calypso-plugins', array( $this, 'plugins_redirect' ) );
```

Comment:
Mixed dashes and underscores here and a separator missing between `top` and `level`. Let's try to be consistent.

Reply:
I've updated the separator to use an underscore where possible. Unfortunately, the dash used in `load-` cannot be changed as it's a part of the action call. The use of `toplevel` also cannot be changed as it is the "type" that comes from core code.

Regarding mixed dashes and underscores. This is how it's done in the original action in `admin.php`. 
https://build.trac.wordpress.org/browser/trunk/wp-admin/admin.php#L192

Reply:
Thanks for the clarification. Yes, that's unfortunate. :smile: